src/adapters/base.ts

```diff
@@ -203,6 +203,10 @@ export abstract class AdapterBase implements Adapter {
         value: any,
         opts?: GetOptions,
     ): Promise<T>;
+    public abstract count<T>(
```

Comment:
Adding this without also adding it to arango technically breaks the arango adapter.

If you don't want add it to arango yourself then you can implement it but just `throw new Error('not implemented')` inside of it to prevent typescript from complaining